collections/_testimonials/24-LuXDEM.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Evaluation of erosion phenomena in a nozzle for Abrasive Water Jet Cutting"
+author: "Prasad ADHAV"
```

Comment:
```suggestion
author: "Prasad Adhav"
```
Just to be consistent with the other posts.

Reply:
Haha, I think the French/Luxembourgish way of writing the names. Now it's muscle memory.